app/js/questions_controller.js

```diff
@@ -120,6 +122,22 @@
     }
   }
 
+  function load_top_helpers() {
```

Comment:
Can you move this into a new `home_controller.js`. `questions_controller.js` is used in three pages and we shouldn't load this info except on the home page.